components/molecules/AlgoliaResults/Facets/Authors.js

```diff
@@ -0,0 +1,27 @@
+import PropTypes from 'prop-types'
+import CustomRefinementList from '../Refinements/CustomRefinementList'
```

Comment:
@dcooney Imports should always use the alias syntax. `@/components/AlgoliaResults/ etc...`

Reply:
Are we good with adding additional alias? Otherwise, some of import statements will get really long.
`@import Teample from '@/components/molecules/AlgoliaSearch/Refinements/CustomRefinementList/'`

Reply:
🤔  Hadn't considered that. This is fine then 👍🏻